src/components/Article.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import '../App.scss';
+
+export const Article = ({ title, date, text }) => (
```

Comment:
Why did you import app style file for the article? Can you create a new style file with just article styles?